activemodel/lib/active_model/errors.rb

```diff
@@ -74,6 +75,28 @@ class Errors
     #     # Will yield <#ActiveModel::Error attribute=name, type=too_short,
     #                                       options={:count=>3}>
     #   end
+
+    ##
+    # :method: clear
+    #
+    # :call-seq: clear
+    #
+    # Clears all errors.
+
+    ##
+    # :method: empty?
+    #
+    # :call-seq: empty?
+    #
+    # Returns true if there are no errors.
+
+    ##
+    # :method: size
+    #
+    # :call-seq: size
+    #
+    # Returns number of errors.
+
     def_delegators :@errors, :each, :clear, :empty?, :size, :uniq!
```

Comment:
This PR omits the documentation for `uniq!`, but I think that's fine.  It's not clear to me that `uniq!` was meant to be public.  It seems to have been added as an implementation detail in 86620cc3aa8e2630bc8d934b1a86453276b9eee9.